docs/source/endpoints/rules.md

```diff
@@ -0,0 +1,168 @@
+---
+myst:
+  html_meta:
+  "description": "Available content rules in a Plone site can be listed and queried by accessing the /@content-rules endpoint on any context. Access requires the ManageContentRules permission."
+  "property=og:description": "Available content rules in a Plone site can be listed and queried by accessing the /@content-rules endpoint on any context. Access requires the ManageContentRules permission."
+  "property=og:title": "Content Rules"
+  "keywords": "Plone, plone.restapi, REST, API, ContentRules, Rules"
+---
+
+(content-rules)=
+
+# Content Rules
+
+```{note}
+These docs are generated by code tests, therefore you will see some `test` rules appear here.
+```
+
+Available content rules in a Plone site can be listed and queried by accessing the `/@content-rules` endpoint in any context.
+Access requires the `ManageContentRules` permission.
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/rules_get.req
+```
```

Comment:
The tabs exploded throughout the page. This might be due to the lack of indentation for the `html_meta` keys.

![Screen Shot 2022-09-28 at 12 36 03 PM](https://user-images.githubusercontent.com/102112/192872921-2ce7a65c-b5ee-464f-9397-565958416208.png)

Reply:
These still do not display correctly in the Preview on Netlify. I am not sure why.

I pulled down this PR to troubleshoot, and found two other bugs that I fixed in two other commits. When I do a `make docs-livehtml`, then these display just fine. `¯\_(ツ)_/¯`. I'm going to resolve this conversation.